src/client/datascience/utils.ts

```diff
@@ -62,9 +62,7 @@ export function translateCellToNative(
         return {
             index: 0,
             metadata: new NotebookCellMetadata().with({
-                executionOrder: cell.data.execution_count as number,
-                hasExecutionOrder: true,
-                runState: translateCellStateToNative(cell.state)
+                hasExecutionOrder: true
```

Comment:
This stuff was being returned for Gather, when we fire events with the information of the cell.
Now we can use VS Code API instead of our wrappers.